dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
This makes sense but I'm unclear if the OWL API will correctly interpret these as anonymous.

Note that it's possible that removing the anonymous distinction may not have much of a negative effect downstream.  Right now I see it primarily being used to query out variants and setting clique leaders.  A lot of this could be fixed by re-modelling our g2p and g2d associations, as suggested in #380.